### ISSUE TITLE
Un-align "equals" on assignations Cop

### ISSUE THREAD

**Issue by pablocrivella** [rubocop-hq/rubocop#7269]
I'm not sure if it exists but I could not find find an Cop that would convert this:

```ruby
foo       = 1
bar       = 2
foobar    = 3
foobarbas = 4
``` 
to this:

```ruby
foo = 1
bar = 2
fopbaar = 3
foobarbas = 4
```

I don't have strong arguments against the 1st option other than sometimes looks really ugly (subjective), it's width inefficient and you have to remember to update the other assignations.

**Comment by bbatsov** [comment]
I think `Layout/ExtraSpacing` accounted for this if you set `AllowForAlignment` to `false`. This cop's documentation needs some love. :-)

**Comment by pablocrivella** [comment]
I can confirm this is working :) thanks a lot for your answer!
Also could you point me to the where this doc update should be added? Maybe I can find some time to introduce a PR for this

**Comment by buehmann** [comment]
See https://docs.rubocop.org/en/stable/development/#documentation

https://github.com/rubocop-hq/rubocop/blob/69bce3fd009a4c2723f3f52e10c3fc54f826df0e/lib/rubocop/cop/layout/extra_spacing.rb#L8-L32
